kie-remote/kie-remote-services/src/main/java/org/kie/remote/services/rest/jaxb/DynamicJaxbContext.java

```diff
@@ -86,27 +70,22 @@
 
  * <b>In all cases, please preserve existing tests when modifying this class!</b>
  */
-@ApplicationScoped
+
 public class DynamicJaxbContext extends JAXBContext {
 
     private static final Logger logger = LoggerFactory.getLogger(DynamicJaxbContext.class);
 
     // The contextsCache needs to be a ConcurrentHashMap because parallel operations (involing *different* deployments) can happen on it.
-    private static ConcurrentHashMap<String, JAXBContext> contextsCache = new ConcurrentHashMap<String, JAXBContext>();
+    static ConcurrentHashMap<String, JAXBContext> contextsCache = new ConcurrentHashMap<String, JAXBContext>();
```

Comment:
I disagree with this, seeing as the `contextsCache` field is being used more in the `DynamicJaxbContext` class than in any other class. With regards to clarity of code, it makes more sense to keep it here.